R/les_kb_v2.R

```diff
@@ -27,30 +27,30 @@ les_kb_oqr_v2 = function(adresse, valider = TRUE) {
   # FIXME! - Se JIRA-sak https://issuetracker.helsenord.no/browse/ABN-372 for hvordan dette løses fremover.
   til_dato_apo = c("maksintervall_start_dato", "maksintervall_slutt_dato")
 
-  kb_oqr = mutate_at(kb_oqr,
-    til_desimal,
-    konverter_tekst,
+  kb_oqr = mutate(kb_oqr,
+    across(all_of(til_desimal),
+    ~ konverter_tekst(.x,
```

Comment:
Flytt eventuelt `across(all_of(` til linja over, og bruk argumentnamn `.fns` for den anonyme funksjonen.